app/views/common/devise/mailer/reset_password_instructions.en.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <p>Someone has requested a link to change your password, and you can do this through the link below.</p>
 
-<p><%= link_to 'Change my password', edit_password_url(@resource, :reset_password_token => @resource.reset_password_token) %></p>
+<p><%= link_to 'Change my password', "#{RunCoCo::Application.config.iframe_parent_domains.first}/portal/collections/world-war-I/contribute#action=reset_password&reset_password_token=#{@resource.reset_password_token}" %></p>
```

Comment:
Could you move this into a v2.1 theme-specific override in case we want to let cataloguers do this on the 1418 site itself, with a different theme?